docs/core/additional-tools/dotnet-svcutil-guide.md

```diff
@@ -18,12 +18,19 @@ The **dotnet-svcutil** tool is an alternative option to the [**WCF Web Service R
 
 ## Prerequisites
 
+# [dotnet-svcutil 2.x](#tab/dotnetsvcutil2x)
```

Comment:
We'll get warnings when try to push the commits to production later on if we don't use the pre-approved values.

```suggestion
# [.NET Core 2.x](#tab/netcore2x)
```

Reply:
@mairaw We want to show that the usage is different depending on which version of the tool you're using, not which version of .NET you have. Changing these tab names to the .NET version doesn't really make sense. The dotnet-svcutil tool is not shipped with the .NET Core SDK and the version numbers are not related. You can use dotnet-svcutil 1.x with only .NET Core 2.x installed. 

Will these warnings be a problem? Should we not be using tabs for this and use a different approach for the older version of the tool?

Reply:
Gotcha. Let's request them to whitelist for now and see what happens. Usually, we don't go this granular on a tool name for the tabs. They're usually large, common values like .NET Core versions, OS names, language, etc.

We try to keep our reports clean of warnings. 

Otherwise, we can come up with a different solution on how to present this but we can publish this as-is for now.